src/components/BoxForm/DropZone/DropZone.jsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/prop-types */
+import React, { useMemo, useState, useEffect } from 'react';
+import { useDropzone } from 'react-dropzone';
+import { CloseIcon } from '@chakra-ui/icons';
```

Comment:
Can we remove the close-icon.png if we are not using it? Same with the imageIcon.png - I don't see these used anywhere, but correct me if I'm wrong.